examples/cucumber/src/test/java/org/testcontainsers/examples/Stepdefs.java

```diff
@@ -0,0 +1,57 @@
+package org.testcontainsers.examples;
+
+import cucumber.api.java.After;
+import cucumber.api.java.Before;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.testcontainers.containers.BrowserWebDriverContainer;
+
+import java.io.File;
+import java.util.List;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.testcontainers.containers.BrowserWebDriverContainer.VncRecordingMode.RECORD_ALL;
+
+public class Stepdefs {
```

Comment:
Please also integrate with `TestLifecycleAware` interface and make it record video after a test failure (by calling `afterTest` with non-empty `throwable` argument

Reply:
added